### ISSUE TITLE
feat: controls

### ISSUE THREAD

**Issue by SassNinja** [scaleflex/js-cloudimage-360-view#7]
I need some controls because dragging is not the best solution for my use case (360 view within a modal). I've checked the source code but seems there's no hook for an easy extension.

So I ended up enabling `keys` and then dispatching the appropriate keyboard events on clicking/holding down my controls (prev/next) but this doesn't feel good and more like a hack.

What do you think about adding a controls feature to this plugin?

Possible markup could be:
```html
<div class="cloudimage-360">
  <button class="cloudimage-360-prev">Prev</button>
  <button class="cloudimage-360-next">Next</button>
</div>
```

**Comment by dzmitry-stramavus** [comment]
@SassNinja, hey

yes, it's a good feature to have. We added it to our roadmap.

**Comment by dzmitry-stramavus** [comment]
@SassNinja , hey!

available in **v2.1.0**

**doc**: https://github.com/scaleflex/js-cloudimage-360-view#-controls
**code sandbox**: https://codesandbox.io/s/js-cloudimage-360-view-361eb?fontsize=14

**Comment by SassNinja** [comment]
@dzmitry-stramavus awesome!

Looks great, thanks
